Gems/ROS2/Code/Source/VehicleDynamics/AxleConfiguration.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/string/string.h>
+
+namespace VehicleDynamics
+{
+    //! An axle has one or more wheels attached. Axle configuration is abstracted form the physical object
+    class AxleConfiguration
+    {
+    public:
+        AZ_TYPE_INFO(AxleConfiguration, "{75BD5FEA-EADE-4371-8B2C-96F05A886BEB}");
+        AxleConfiguration() = default;
+        static void Reflect(AZ::ReflectContext* context);
+
+        //! Helper functions for wheel entities. If there is only one wheel, same value is returned from both.
+        //! If no wheels are set, this will return invalid EntityId (check with IsValid())
```

Comment:
Doxygen tags will associate with a single entry. These should not use them since they are describing a group of functions
```suggestion
        // Helper functions for wheel entities. If there is only one wheel, same value is returned from both.
        // If no wheels are set, this will return invalid EntityId (check with IsValid())
```

Reply:
Done